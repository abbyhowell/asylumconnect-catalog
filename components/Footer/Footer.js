import React from 'react';
import { Link } from '../../routes';
import { Container, Typography } from '@material-ui/core';

const Footer = () => (
  <footer>
    <Container maxWidth="lg">
      <Typography variant="h5">Asylum Connect</Typography>
    </Container>
  </footer>
);

export default Footer;
