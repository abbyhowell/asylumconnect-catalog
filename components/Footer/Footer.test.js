import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

it('renders correctly', () => {
  const tree = shallow(<Footer />).html();
  expect(tree).toMatchSnapshot();
});
