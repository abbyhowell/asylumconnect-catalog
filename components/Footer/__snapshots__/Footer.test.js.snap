// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `"<div class=\\"MuiContainer-root MuiContainer-fixed MuiContainer-maxWidthLg\\"><p>Asylum Connect</p></div>"`;
